packages/contracts-bedrock/src/safe/SaferSafes.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Safe
+import { GnosisSafe as Safe } from "safe-contracts/GnosisSafe.sol";
+
+// Safe Extensions
+import { LivenessModule2 } from "./LivenessModule2.sol";
+import { TimelockGuard } from "./TimelockGuard.sol";
+
+/// @title SaferSafes
+/// @notice Combined Safe extensions providing both liveness module and timelock guard functionality
+/// @dev This contract can be enabled simultaneously as both a module and a guard on a Safe:
+///      - As a module: provides liveness challenge functionality to prevent multisig deadlock
+///      - As a guard: provides timelock functionality for transaction delays and cancellation
```

Comment:
Would suggest a bit more docs on installation options to answer questions like:
- If you don't want the timelock, should I simply not install this as a guard, or should I still install it but with zero delay? 
- What if you want the timelock but not the liveness? I think we should just say that is not supported and untested because I don't think we've thought about that much, and everyone should have a liveness guard